src/components/Tips/Tips.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+
+import { setNotificationAction } from 'actions';
+
+import './Tips.scss';
+
+interface ITipsProps {
+    str: string;
+    tips: string;
+    className?: string;
+}
+
+export const Tips =
+    ({ str, tips, className }: ITipsProps) => {
+        const dispatch = useDispatch();
+        const onClick = () => {
+            dispatch(setNotificationAction({ text: tips }));

```

Comment:
Ne je li to prěmnogo tolsto – pokazyvati "toast notification"?

Reply:
Myslim, da najlěpše bude pokazati někako kolo s "i" (ℹ️ ), kde pri natiskanju abo navodženju kursora bude někaky "popover"... 🗯️